metrics/log/log.go

```diff
@@ -23,8 +24,23 @@ func (lr *logRecorder) Handler() http.Handler {
 	return nil
 }
 
-func mustString(s string, _ error) string {
-	return s
+func simpleGoValue(node ipld.Node) interface{} {
+	var val interface{}
+	var err error
+	switch kind := node.Kind(); kind {
+	case ipld.Kind_String:
+		val, err = node.AsString()
```

Comment:
can these be simplified to `return node.String()`

Reply:
hm, I'm pretty sure that's not an IPLD method. it looks like we define it for Status, but that's for an Int that we want to behave like a human-friendly string.

if that's something we want to do for logging (the answer is probably "yes"), then for Status fields we could call the String method instead of this func here.

And I'm actually just realising that its String method might not work in practice. It uses a pointer-based map, so if we do an IPLD codec decode, we'd end up with different pointers and... no useful string :)

Reply:
I've been doing that through a bunch of this code and it seems to work, e.g 
https://github.com/filecoin-project/dealbot/blob/main/controller/state/statedb.go#L492